<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Andrey Rijov</author>
 <class>Folder_Sharing_Widget</class>
 <widget class="QWidget" name="Folder_Sharing_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Folder Sharing Widget</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="5">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <layout class="QGridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="Label_View_Mode">
       <property name="text">
        <string>View Mode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="TB_Icon_Mode">
         <property name="toolTip">
          <string>Icon Mode</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/images.qrc">
           <normaloff>:/view_icon.png</normaloff>:/view_icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="TB_List_Mode">
         <property name="toolTip">
          <string>List Mode</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/images.qrc">
           <normaloff>:/view_detailed.png</normaloff>:/view_detailed.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="Label_Manage_Folders">
       <property name="text">
        <string>Remove Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="TB_Remove_Folder">
         <property name="toolTip">
          <string>Remove from shared folders list</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/images.qrc">
           <normaloff>:/remove.png</normaloff>:/remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="Label_Add_Folders">
       <property name="text">
        <string>Add Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="TB_Add_Folder">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Add to shared folders list</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resources/images.qrc">
           <normaloff>:/open-folder.png</normaloff>:/open-folder.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="Label_Folders_List">
       <property name="text">
        <string>Folder list:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="Folders_List">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="flow">
        <enum>QListView::LeftToRight</enum>
       </property>
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Information">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>How to mount the shared folder on the guest (9p fs support is required on the guest):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Connected_To">
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAdd_Folder">
   <property name="icon">
    <iconset>
     <normaloff>:/images/open-folder.png</normaloff>:/images/open-folder.png</iconset>
   </property>
   <property name="text">
    <string>Add Folder</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset>
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
   <property name="iconText">
    <string>Properties</string>
   </property>
   <property name="toolTip">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset>
     <normaloff>:/images/button_cancel.png</normaloff>:/images/button_cancel.png</iconset>
   </property>
   <property name="text">
    <string>Remove From List</string>
   </property>
  </action>
  <action name="actionList_Mode">
   <property name="icon">
    <iconset>
     <normaloff>:/images/view_detailed.png</normaloff>:/images/view_detailed.png</iconset>
   </property>
   <property name="text">
    <string>List Mode</string>
   </property>
  </action>
  <action name="actionIcon_Mode">
   <property name="icon">
    <iconset>
     <normaloff>:/images/view_icon.png</normaloff>:/images/view_icon.png</iconset>
   </property>
   <property name="text">
    <string>Icon Mode</string>
   </property>
  </action>
  <action name="actionQuick_Format">
   <property name="icon">
    <iconset>
     <normaloff>:/images/reload.png</normaloff>:/images/reload.png</iconset>
   </property>
   <property name="text">
    <string>Quick Format</string>
   </property>
  </action>
  <action name="actionAdd_Folder2">
   <property name="icon">
    <iconset>
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Add Folder</string>
   </property>
   <property name="iconText">
    <string>Add Folder</string>
   </property>
   <property name="toolTip">
    <string>Add Folder</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>TB_Remove_Folder</tabstop>
  <tabstop>TB_Icon_Mode</tabstop>
  <tabstop>TB_List_Mode</tabstop>
  <tabstop>Folders_List</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>TB_Icon_Mode</sender>
   <signal>clicked()</signal>
   <receiver>actionIcon_Mode</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TB_List_Mode</sender>
   <signal>clicked()</signal>
   <receiver>actionList_Mode</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TB_Add_Folder</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd_Folder</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TB_Remove_Folder</sender>
   <signal>clicked()</signal>
   <receiver>actionRemove</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
